Migrating-AU-to-Chocolatey-AU.md

```diff
@@ -0,0 +1,27 @@
+# Migrating From AU To Chocolatey AU
+
+With the [`1.0.0` release] of Chocolatey AU, there is a need to migrate to the new package name.
+
+How you migrate from AU to Chocolatey AU will depend on how you used AU in your repository. Below are some scenarios, if you encounter a scenario not yet covered, please reach out in the [Chocolatey Community].
+
+## Your Repository Currently Clones The [Original AU Repository]

```

Comment:
Let's not start making use of links in the header, while it will work in most cases, there can be times it won't work properly.

Reply:
That's fair. I've removed it from the headings.